core/src/main/scala/akka/persistence/cassandra/query/EventsByTagFetcher.scala

```diff
@@ -113,6 +113,9 @@ private[query] class EventsByTagFetcher(
           else
             highestOffset = offs
 
+          // FIXME reproducer of issue #206
+          // Thread.sleep(30)
```

Comment:
the problem is when replaying the last time bucket takes long time, so that the `LookingForDelayed` mode is triggered (over and over again)